CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Project Pythia Cookbook Contributor's Guide
+
+Project Pythia Cookbooks are collections of more advanced and domain-specific example
+workflows building on top of [Pythia Foundations](https://foundations.projectpythia.org/landing-page.html). 
+They are [geoscience](https://en.wikipedia.org/wiki/Earth_science)-focused
+and should direct the reader towards the [Foundations material](https://foundations.projectpythia.org/landing-page.html) for any required
+background knowledge. 
+
+The following is a step-by-step guide to getting your cookbook idea
+hosted on the [Project Pythia Cookbooks gallery](https://projectpythia.org/cookbook-gallery.html).
+
+1. Use the template
+    1. On https://github.com/ProjectPythiaTutorials/cookbook-template, click "Use this template"
+    1. Create the repo under your account with a descriptive name, followed by `-cookbook` (e.g. `hydrology-cookbook`, `hpc-cookbook`, `cesm-cookbook`, etc.)
+    1. In the settings, enable GitHub Pages and make the source `gh-pages/(root)`
```

Comment:
```suggestion
    1. Your browser should be directed to the newly created repository under your GitHub account. Under Settings, enable GitHub Pages by changing the Source from `none` to `main` and clicking on `Save`
```

Reply:
Done, but `main` should be `gh-pages`

Reply:
Can you confirm this? I'm pretty sure when I went through the process the pull-down gave me the option of selecting 'none' or 'main'. Thanks.

Reply:
Now that I look at it, there's an option "Include all branches" when using the template that would need to be chosen to get the `gh-pages` branch as well. I'll add that to the guide.